.github/workflows/application-signals-python-e2e-ec2-test.yml

```diff
@@ -32,11 +32,12 @@ env:
   METRIC_NAMESPACE: AppSignals
   LOG_GROUP_NAME: /aws/appsignals/generic
   ADOT_WHEEL_NAME: ${{ inputs.staging_wheel_name }}
-  TEST_RESOURCES_FOLDER: /home/runner/work/aws-application-signals-test-framework/aws-application-signals-test-framework
 
 jobs:
   python-e2e-ec2-test:
     runs-on: ubuntu-latest
+    container:
+      image: public.ecr.aws/h6o3z5z9/aws-application-signals-test-framework-workflow-container:latest
```

Comment:
Are there any tools/packages that the python test uses that can be added to the Dockerfile we created in the test framework?

Reply:
There is no extra need for python. The E2E test steps are very similar as Java, all the python related installation are added through terraform instead of workflow.